Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -7136,6 +7143,9 @@ on revIDESaveStack pStackID
    
    lock messages
    
+   local tStackFilename
+   put the effective filename of stack tStackName into tStackFilename
+   
    local tStackFileVersion
    
    if the cPreserveStackVersion of stack "revPreferences" is true and the cREVGeneral["stackfileversion"] of stack tStackName is not empty then
```

Comment:
@livecodeali `cREVGeneral["stackfileversion"]` can be changed to `revIDEStackFileVersion()` I think. Or do we for some reason want the user to be able to save the stack with non-ide commands to a different format and then revert the stack when saving via the IDE command?